server/routes/changeEstablishment.test.ts

```diff
@@ -174,6 +174,7 @@ describe('POST /change-establishment', () => {
           username: 'user1',
           operationId: undefined,
         })
+        // @TODO should also check setActiveCaseLoad is called (awaiting VB-1430)
```

Comment:
Ideally would check case load gets set here but got bogged down trying to do so because of the way `MockUserService` is used in `appSetup.ts` - figured this would be better sorted out once work to refactor to use a services container is done (VB-1430)